ipam/allocator.go

```diff
@@ -70,6 +70,8 @@ func NewAllocator(lcDs, glDs datastore.DataStore) (*Allocator, error) {
 		}
 	}
 
+	a.checkConsistency(localAddressSpace)
```

Comment:
Why are we not doing it for `globalAddressSpace`. The same bug exists there too.